doc/systest_basics.rst

```diff
@@ -0,0 +1,288 @@
+histograms, optionally using optimized bin widths.  You can pass a one-dimensional dataset to
+display its histogram; a list of one-dimensional datasets to generate histograms on the same axes;
+a formatted array (see :doc:`data`) and a field to generate a histogram for that field; or a list of
+formatted arrays and a field or list of fields to generate multiple histograms on the same axes.
+
+To operate on a data vector:
+
+>>> histogram_sys_test_vector = stile.HistogramSysTest()
+
+To operate on a formatted array,
+
+>>> histogram_sys_test_field = stile.HistogramSysTest('g1')
+
+or pass the field at runtime:
+
+>>> results = histogram_sys_test_vector(data, field='g1')
+
+To operate on multiple data vectors:
+
+>>> results = histogram_sys_test_vector([data, data2])
+
+or on multiple formatted arrays:
+
+>>> results = histogram_sys_test_field([data, data2])
+
+or
+
+>>> results = histogram_sys_test_vector([data, data2], field='g1').
+
+To plot multiple fields from the same array, you'll need to pass the same array multiple times, due
+to implementation details:
+
+>>> results = histogram_sys_test_vector([data, data], field=['g1', 'g2'])
+
+or of course you can just access those fields yourself:
+
+>>> results = histogram_sys_test_vector([data['g1'], data['g2']]).
+
+By default, the :class:`HistogramSysTest <stile.sys_tests.HistogramSysTest>` makes a histogram with
+50 bins.  You probably want to choose different binning.  You can have Stile pick the binning for you
+based on two different algorithms:
+
+>>> scott_rule_hist = stile.HistogramSysTest(binning_style='scott')  # Scott's rule
+>>> freedman_rule_hist = stile.HistogramSysTest(binning_style='freedman')  # Freedman-Diaconis rule
+
+You can also just set your own number of bins.
+
+>>> ten_bins_hist = stile.HistogramSysTest(nbins=10)
+
+These plots are very configurable--you can add vertical lines, colors, text, and you can also change 
+the appearance of the histogram itself.  See the documentation for :class:`HistogramSysTest 
+<stile.sys_tests.HistogramSysTest>` for more information.
+
+Basic statistical quantities
+----------------------------
+
+The :class:`StatSysTest <stile.sys_tests.StatSysTest>` systematics tests are designed to measure
+basic statistical quantities on a vector of data or on a field from a data array.  To operate on a
+data vector:
+
+>>> stat_sys_test_vector = stile.StatSysTest()
+
+To operate on a formatted array,
+
+>>> stat_sys_test_field = stile.StatSysTest('g1')
+
+or pass the field at runtime:
+
+>>> results = stat_sys_test_vector(data, field='g1')
+
+Currently, a :class:`stile.StatSysTest <stile.sys_tests.StatSysTest>` will compute min, max,
+median, median absolute deviation (MAD), mean, standard deviation, variance, and number of
+objects, and will additionally compute the skew and kurtosis if ``scipy.stats`` can be imported.
+It will also compute percentiles in the data: by default, 1, 2, and 3 sigma values around the
```

Comment:
Perhaps clarify that this is based on the confidence intervals for a Gaussian - i.e., it won't compute sigma and then use that to tell you the 1/2/3-sigma values, but rather uses the numpy percentiles routine to get the values of the data at the percentiles corresponding to 1, 2, and 3 sigma about the median.